src/lib/swapper/swappers/ThorchainSwapper/utils/getLongtailQuote.ts

```diff
@@ -87,6 +94,35 @@ export const getLongtailToL1Quote = async (
     sellAmountIncludingProtocolFeesCryptoBaseUnit: quotedAmountOut.toString(),
   }
 
-  const thorchainQuote = await getL1quote(l1Tol1QuoteInput, streamingInterval)
-  return thorchainQuote
+  const thorchainQuotes = await getL1quote(
+    l1Tol1QuoteInput,
+    streamingInterval,
+    TradeType.LongTailToL1,
+  )
+
+  return thorchainQuotes
+    .mapErr(e => {
+      console.error('Thorchain quote error:', e)
```

Comment:
can possibly remove the string since without the `Logger` we get a stack trace pointing to this file anyway
```suggestion
      console.error(e)
```

Reply:
https://github.com/shapeshift/web/pull/5719/commits/d4723c0584ab8adf1c3f8e826e1d59bc0492b935